### DISCUSSION TITLE
Practical Clean Architecture solution template

### DISCUSSION THREAD

**Discussion by giscus** [CamiloTerevinto/Blog#6] [Announcements]
# Practical Clean Architecture solution template

https://www.camiloterevinto.com/post/practical-clean-architecture-template

**Comment by gabrielsimas** [comment]
Good approach! As my part, in my Application domain, I called your "Domain Layer" like a "Application Layer"... and My API Layer it's a MinimalAPI decoupled by program.cs. And I'm decoupling the Data Infrastructure like a Persistence Layer! But That's just semantics! I liked quite a lot of your approach and I'll give it a try in my next refactoring rounds...
